tests/test_client.py

```diff
@@ -72,6 +73,9 @@ def test_safe_urlencode(self):
             "test=Hello \u2603!&test=Helllo world!",
         )
 
+        # Boolean options for Solr should be in lowercase.
+        self.assertTrue("True" not in safe_urlencode({"group": True}))
```

Comment:
This would pass a bug which encoded it as `group=None` or `group="false"`.

```suggestion
        self.assertEqual("group=true", safe_urlencode({"group": True}))
```